tools/clipper.js

```diff
@@ -55,9 +55,12 @@ async function clipRequest([videoType, videoLink, timestamps, projectName, fileE
 				const archive = archiver('zip', {
 					zlib: { level: 9 },
 				});
+
+				zipFile.on('close', () => {
+					archive.finalize();
+				});
```

Comment:
:LunaGalaxy: 
Should be archive.on('close'), in the callback you need to have all the uploading to Nextcloud stuff. (The stuff that needs to be executed after the archive has been created) 
Move `archive.finalize();` back down again.